crates/sui-indexer-alt-jsonrpc/src/api/transactions/filter.rs

```diff
@@ -394,11 +397,33 @@ async fn from_sequence_numbers(
         .transpose()
         .context("Failed to encode next cursor")?;
 
-    let digests = ctx
-        .pg_loader()
-        .load_many(rows.iter().map(|&seq| TxDigestKey(seq as u64)))
-        .await
-        .context("Failed to load transaction digests")?;
+    // Get digests from the table and retry any of the digests that are not found.
+    let config = &ctx.config().transactions;
+    let retries = config.tx_digest_retry_count;
+    let mut retry_interval =
+        tokio::time::interval(Duration::from_millis(config.tx_digest_retry_interval_ms));
+    let mut keys: Vec<_> = rows.iter().map(|&seq| TxDigestKey(seq as u64)).collect();
+    let mut digests: HashMap<TxDigestKey, StoredTxDigest> = HashMap::new();
+    for _ in 0..retries {
```

Comment:
As per the other comment, this should probably be an inclusive upperbound

```suggestion
    for _ in 0..=retries {
```